Return-path: <hristo@venev.name>
Envelope-to: rene@exactcode.de
Delivery-date: Mon, 08 May 2023 17:41:32 +0000
Received: from a1-bg02.venev.name ([213.240.239.49])
	by exactco.de with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.86_2)
	(envelope-from <hristo@venev.name>)
	id 1pw4rb-0004HW-A8
	for rene@exactcode.de; Mon, 08 May 2023 17:41:32 +0000
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed; d=venev.name;
	s=default; h=Content-Transfer-Encoding:Message-Id:Date:Subject:To:From:
	Content-Type:Reply-To:Sender; bh=33w0CTUjTZH/FbGuypNBdYhdd6ZyEOaBwYbpVeVOPjI=
	; b=MMPrL1qn/hmsGbjZT+IjukIWerHJsW6PV/P7TmhjjfyA+tQ3K/3ojUQHfCvOSM5QnEJ3MFcEf
	fSohDbISyuTIuHV0jpIijHSpabDc4V343TpWXfvuDeV2hqn4mRa898XKy+ppY9kXPcr7uP2wh2iia
	xOiHgiYvkdvpA6+UdgDObBrsIkwrtOIWBUR5EyTCQggjTkkEOhLpwDvfFvZPAScucvJCd+dWOWjx+
	k11wiAKMjkD1MdcMYzOxHlgzU8nVzSK2RIU+g5ptVgBK6N1w+Cooz9QmVIUbjZccnUYq5u1xhp909
	Ge7sU185M4lkScaUCCSUsf0cgTwQOZXwZwnWZPbOrEwjKtlnDT/Z9f8rD/HRasQfuTe+wU2pioYok
	L17dVY8Qs37AeQdUAy+kORN07mR65ony6eYzOryemulkiG9X+bpmrjYiSkwyPmHuDnYDVRagKbMMF
	WWjhS7aULZ1APD5C1Jog64N/wpl7pZni9g9uj+fntYLk0dg2SilqZi2h5AIb+6pY7ckZ0TlF0//W4
	IwQ/2mscaoPZDRcd7lTJFqLHlYtYUbhLHbTO9BZn7s+fe7iOFZjmf/wFcOM5hwJ0LwIXy87LT1pfP
	cojeodAxS/q/FPKHGNlIPGiyBddtLGt8padMyT2mbPRHSFZRmVAWtJ7CLFOBSF3Fbx1tSao=;
Received: from a1-bg02.venev.name ([213.240.239.49] helo=pmx1.venev.name)
	by a1-bg02.venev.name with esmtps
	id 1pw4mx-000rGt-33
	(TLS1.3:TLS_AES_256_GCM_SHA384:256)
	(envelope-from <hristo@venev.name>);
	Mon, 08 May 2023 17:36:43 +0000
Received: from venev.name ([213.240.239.49])
	by pmx1.venev.name with ESMTPSA
	id C9sRICkzWWTnHwMAdB6GMg:T2
	(envelope-from <hristo@venev.name>); Mon, 08 May 2023 17:36:43 +0000
From: Hristo Venev <hristo@venev.name>
To: =?UTF-8?q?Ren=C3=A9=20Rebe?= <rene@exactcode.de>
Cc: t2@t2sde.org,
	Hristo Venev <hristo@venev.name>
Subject: [PATCH 1/2] dt-bindings: hwmon: jedec,spd5118: Add binding
Date: Mon,  8 May 2023 20:35:31 +0300
Message-Id: <20230508173531.204602-2-hristo@venev.name>
X-Mailer: git-send-email 2.40.1
In-Reply-To: <20230508173531.204602-1-hristo@venev.name>
References: <20230508173531.204602-1-hristo@venev.name>
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Spam-Score: 3.1 (+++)

Add device tree bindings for the SPD hub present in DDR5 modules.

Signed-off-by: Hristo Venev <hristo@venev.name>
---
 .../bindings/hwmon/jedec,spd5118.yaml         | 42 +++++++++++++++++++
 1 file changed, 42 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/hwmon/jedec,spd5118.yaml

diff --git a/Documentation/devicetree/bindings/hwmon/jedec,spd5118.yaml b/Documentation/devicetree/bindings/hwmon/jedec,spd5118.yaml
new file mode 100644
index 000000000000..9410ee0cb6b6
--- /dev/null
+++ b/Documentation/devicetree/bindings/hwmon/jedec,spd5118.yaml
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: GPL-2.0-only or BSD-2-Clause
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/hwmon/jedec,spd5118.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: JEDEC JESD300-5B compatible DDR5 SPD hub
+
+maintainers: []
+
+select:
+  properties:
+    compatible:
+      const: jedec,spd5118
+
+  required:
+    - compatible
+
+properties:
+  compatible:
+    const: jedec,spd5118
+
+  reg:
+    maxItems: 1
+
+required:
+  - compatible
+  - reg
+
+additionalProperties: false
+
+examples:
+  - |
+    i2c {
+        #address-cells = <1>;
+        #size-cells = <0>;
+
+        temp-sensor@1a {
+            compatible = "jedec,spd5118";
+            reg = <0x51>;
+        };
+    };
-- 
2.40.1

